### ISSUE TITLE
Site title incorrectly pulled when submitting to Reddit

### ISSUE THREAD

**Issue by IfaqYurmama** [NAVCoin/nav-community-site#18]
![image](https://user-images.githubusercontent.com/33290035/38609298-fb0455ce-3d7d-11e8-9d5d-349a0bf59f8f.png)

Usually it should pull the site title that's also displayed in the tab (
![image](https://user-images.githubusercontent.com/33290035/38609384-354788e6-3d7e-11e8-9a6f-ca401462dcac.png))

**Comment by matt-auckland** [comment]
The info that gets pulled by reddit is stored in the header of each page, but with the current set up I'm not sure if we can modify the header per news post as the header is a sibling partial of the content. @red010b37 @craigmacgregor Can you confirm this?

Where the info is pulled from: https://github.com/NAVCoin/nav-community-site/blob/master/themes/nav-community-v1/layouts/partials/head.html

**Comment by zanuka** [comment]
@matt-auckland @IfaqYurmama @red010b37 @craigmacgregor yeah this can be fixed by creating a partial template for meta elements so things like title, twitter description, share image can be set on a per post basis. Right now it's all being set globally in head.html. good catch!
